src/controllers/hid/hidcontroller.cpp

```diff
@@ -243,13 +245,22 @@ int HidController::close() {
 bool HidController::poll() {
     Trace hidRead("HidController poll");
 
-    int result = hid_read(m_pHidDevice, m_pPollData, sizeof(m_pPollData) / sizeof(m_pPollData[0]));
-    if (result == -1) {
-        return false;
-    } else if (result > 0) {
-        Trace process("HidController process packet");
-        QByteArray outData(reinterpret_cast<char*>(m_pPollData), result);
-        receive(outData, mixxx::Time::elapsed());
+    int result = 1;
+    auto loopStartTime = mixxx::Time::elapsed();
+    while (result > 0) {
+        // Failsafe in case the script takes too long. This can happen if a controller constitutively spams
```

Comment:
Do you mean "continuously"?

Reply:
Those are synonyms.

Reply:
I've never heard this word either, can we keep the language a little simpler for all of our non-native speakers?

Reply:
I looked up "constitutively" and it seems to mostly be used in the context of biochemistry and biology so I'll change this to the more common synonym.